client/src/app/components/panelAdmin/dragndrop/dragndrop.component.ts

```diff
@@ -45,46 +41,33 @@ export interface IDialogData {
 @Component({
   selector: 'app-dialog-overview-test',
   templateUrl: 'dialog-overview-test.html',
-  styleUrls: ['./dialog-overview-test.scss']
+  styleUrls: ['./dialog-overview-test.scss'],
 })
 export class DialogOverviewTestComponent implements OnInit {
   dataPopup;
   prev = false;
   constructor(
     public dialogRef: MatDialogRef<DialogOverviewTestComponent>,
-    @Inject(MAT_DIALOG_DATA) public data: IDialogData
+    @Inject(MAT_DIALOG_DATA) public data: IDialogData,
   ) {
     if (data) {
       this.dataPopup = data;
       this.prev = true;
     }
-    
   }
 
   ngOnInit() {
     console.log('this.dataPopup : ', this.dataPopup);
   }
 
-  onNoClick(): void {
-    this.dialogRef.close();
-  }
-  onClick(): void {
-    console.log(this.data);
-    this.dialogRef.close();
-  }
-  refreshComponent($event): void {
```

Comment:
@Adrien-Leteinturier  remove refreshComponent in template dialog-overview-test.html, tag <app-test ..> </app-test> build fail